src/store/indexeddb-local-backend.js

```diff
@@ -239,42 +223,39 @@ LocalIndexedDBStoreBackend.prototype = {
     },
 
     /**
-     * Persist a list of User objects. Users with the same 'userId' will be
-     * replaced.
-     * @param {User[]} users An array of users
+     * Persist a list of [user id, presence event] they are for.
+     * Users with the same 'userId' will be replaced.
+     * Prfesence events should be the event in its raw form (not the Event
+     * object)
+     * @param {Object[]} tuples An array of [userid, event] tuples
```

Comment:
Happy to change if you prefer. Tuples will be a little smaller and so go over postMessage faster. For 2 things I personally think a tuple is fine. It's an internal interface so trivial to change to objects if we want.